app/models/server/raw/Rooms.js

```diff
@@ -76,6 +76,26 @@ export class RoomsRaw extends BaseRaw {
 		return this.find(query, options);
 	}
 
+	findByTeamId(teamId, options = {}) {
+		const query = {
+			teamId,
+		};
+
+		return this.find(query, options);
+	}
+
+	findByTeamIdAndUserId(uid, teamId, options = {}) {
+		const query = {
+			teamId,
+			$or: [
+				{ t: 'c' },
+				{ 'u._id': uid },
```

Comment:
on *rooms* collection the field `u._id` represents the room creator, but looks like this is not what you is trying to find, right?